src/scrabble.js

```diff
@@ -1,5 +1,191 @@
+        newWord[i].includes('E') ||
+        newWord[i].includes('I') ||
+        newWord[i].includes('O') ||
+        newWord[i].includes('U') ||
+        newWord[i].includes('L') ||
+        newWord[i].includes('N') ||
+        newWord[i].includes('R') ||
+        newWord[i].includes('S') ||
+        newWord[i].includes('T') ||
+        newWord[i].includes('a') ||
+        newWord[i].includes('e') ||
+        newWord[i].includes('i') ||
+        newWord[i].includes('o') ||
+        newWord[i].includes('u') ||
+        newWord[i].includes('l') ||
+        newWord[i].includes('n') ||
+        newWord[i].includes('r') ||
+        newWord[i].includes('s') ||
+        newWord[i].includes('t')
+      ) {
+        sum += 1
+      }
+      if (
+        newWord[i].includes('D') ||
+        newWord[i].includes('G') ||
+        newWord[i].includes('d') ||
+        newWord[i].includes('g')
+      ) {
+        sum += 2
+      }
+      if (
+        newWord[i].includes('B') ||
+        newWord[i].includes('C') ||
+        newWord[i].includes('M') ||
+        newWord[i].includes('P') ||
+        newWord[i].includes('b') ||
+        newWord[i].includes('c') ||
+        newWord[i].includes('m') ||
+        newWord[i].includes('p')
+      ) {
+        sum += 3
+      }
+      if (
+        newWord[i].includes('F') ||
+        newWord[i].includes('H') ||
+        newWord[i].includes('V') ||
+        newWord[i].includes('W') ||
+        newWord[i].includes('Y') ||
+        newWord[i].includes('f') ||
+        newWord[i].includes('h') ||
+        newWord[i].includes('v') ||
+        newWord[i].includes('w') ||
+        newWord[i].includes('y')
+      ) {
+        sum += 4
+      }
+      if (newWord[i].includes('K') || newWord[i].includes('k')) {
+        sum += 5
+      }
+      if (
+        newWord[i].includes('J') ||
+        newWord[i].includes('X') ||
+        newWord[i].includes('j') ||
+        newWord[i].includes('x')
+      ) {
+        sum += 8
+      }
+      if (
+        newWord[i].includes('Q') ||
+        newWord[i].includes('Z') ||
+        newWord[i].includes('q') ||
+        newWord[i].includes('z')
+      ) {
+        sum += 10
+      }
+    }
+    return sum
+  }
+}
+
+function doubleLetter(aWord) {
+  const regex1 = /{[A-Z|a-z]}/g
+  const found1 = aWord.match(regex1)
+  const doubleLetter = normalScore(found1) * 2
+  return doubleLetter
+}
+
+console.log('Double letter test', doubleLetter('{A}{A}'))
+
+function trippleLetter(aWord) {
+  const regex2 = /\[[A-Z | a-z]]/g
+  const found2 = aWord.match(regex2)
+  const trippleLetter = normalScore(found2) * 3
+  return trippleLetter
+}
+
+function normalLetter(aWord) {
+  const split = aWord.replaceAll(/{[A-Z | a-z]}/g, '')
+  const split2 = split.replaceAll(/\[[A-Z | a-z]]/g, '')
 
+  console.log('split3', split2.toString())
+  console.log('First Normal Score', normalScore(split2))
+  return normalScore(split2)
 }
 
+function doubleWords(aWord5) {
+  const myLength = aWord5.length
```

Comment:
Why does this suddenly change from aWord to aWord5?

Reply:
Honestly, I was so desperate to get the test passed that I just tried changing that variable name. Thanks for spotting that. Will change it.